watch-deployment.js

```diff
@@ -0,0 +1,95 @@
+const axios = require("axios").default;
+const core = require("@actions/core");
+
+let watchDeployment = function (
+  harness_api_url,
+  harness_ui_url,
+  harness_api_key,
+  options = {}
+) {
+  const { waitBetween, timeLimit } = Object.assign(
+    { waitBetween: 10, timeLimit: 1200 },
```

Comment:
20mins is probably enough for our harness deploys but we may need to expose this as an input to the action at some point.